python/pyproject.toml

```
@@ -0,0 +1,15 @@
+[build-system]
+requires = ["setuptools>=42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "prd"
```

<https://pypi.org/project/prd> already taken so pip release will fail.

---

It'll become https://pypi.org/project/petsird/

---

ok I've claimed https://pypi.org/project/petsird just to ensure nobody else does :)

```suggestion
name = "petsird"
```

---

We'll also need to change "PRD" to "petsird" in https://github.com/ETSInitiative/PRDdefinition/blob/main/model/_package.yml to ensure the folder name matches the name in `pyproject.toml`

---

We should first merge #22, then tackle #27 (lots of renames), then merge that back here, only then change to petsird. #27 implies far too many changes to do it in a PR on Pypi 😄